ui-tests/cypress/e2e/administration/language_and_labels/system_labels.cy.js

```diff
@@ -363,6 +383,31 @@ describe('Labels', () => {
     });
 
   });
+  const getKeysFromLabelsJSON = () => {
+    cy.fixture('data/labels.json').then(labels => Object.values(labels).map(label => label.key)).as('keysFromJSON');
```

Comment:
There's no need to create an alias if you just have to return it the line immediately after